samples/gcp-cloud-calls/index.js

```diff
@@ -0,0 +1,18 @@
+
+const { App } = require("core");
+const { GcpModule, GcpFunctionCloudService } = require("gcp");
+const { HTTPBindingMiddleware }  = require("core");
```

Comment:
this should be : const  { App, HTTPBindingMiddleware } = require("core"),  to avoid use one code line.

Reply:
done